lib/utilities/index.js

```diff
@@ -12,7 +12,7 @@ PlatformEventEmitter.addListener('DJIEvent', evt => {
   DJIEventSubject.next(evt);
 });
 
-// DJIEventSubject.subscribe(evt => console.log(evt));
+DJIEventSubject.subscribe(evt => console.log(evt));
```

Comment:
Think you forgot to comment this out again

Reply:
I did, thank you! 🙈